package.json

```diff
@@ -27,24 +27,25 @@
     "url": "https://github.com/mattboutet/tandy/issues"
   },
   "dependencies": {
-    "boom": "3.x.x",
-    "call": "4.x.x",
-    "hapi": "16.x.x",
-    "hoek": "4.x.x",
+    "boom": "6.x.x",
+    "bounce": "^1.2.0",
```

Comment:
is this used anywhere? my bad if I just missed something?

Reply:
Good catch - I pulled it in, then never used it.  Burned by npm v5